Week1/homework/js-exercises/ex5.js

```diff
@@ -0,0 +1,25 @@
+//Exercise 5: Round a number and log it
+//Declare variable z and assign the number 7.25 to it
+let z = 7.25;
+//log the value of z
+console.log(z);
+//Declare variable a and assign z.round value
+let a = Math.round(z);
+//log the value of a
+console.log(a);
+//compare the two values and store the highest of the two in a new variable
+if (a > z) {
+  let x = a;
+} else {
+  let x = z;
+  console.log(x);
+}
+//console.log
+function compare(a, z) {
+  if (a > z) {
+    return a;
+  } else {
+    return z;
+  }
+}
+console.log(`this this the highest ${compare(a, z)}`);
```

Comment:
Aha, I see what you did here, but there is an easier way to do this, with [Math.max](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max).

Reply:
thanks for the link, it was really easier, Math way have been added locally.